data/mehmetsaidozsoy.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Mehmet Said ÖZSOY",
+  "displayStatsPublic": true,
```

Comment:
Also, remove the `displayStatsPublic` key from your json. We recently removed the public stats feature, so this property is no longer needed. 

```suggestion
```